### PR TITLE
[Improve] flink shims regexp pattern imporvement

### PR DESCRIPTION
**Pull request by wolfboys**
[Improve] flink shims regexp pattern imporvement

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @lvshaokang PTAL, thx

File: streampark-flink/streampark-flink-proxy/src/main/scala/org/apache/streampark/flink/proxy/FlinkShimsProxy.scala [hunk 1]
```diff
@@ -50,11 +48,29 @@ object FlinkShimsProxy extends Logger {
     s"streampark-(.*)_$scalaVersion-(.*).jar",
     Pattern.CASE_INSENSITIVE | Pattern.DOTALL)
 
+  private[this] def getShimsLibPattern(flinkVersion: FlinkVersion): Pattern = {
```
Comment (lvshaokang) [inline streampark-flink/streampark-flink-proxy/src/main/scala/org/apache/streampark/flink/proxy/FlinkShimsProxy.scala:51]: some minor problems, can add some tests to prove effect

**Review by lvshaokang — COMMENTED** [review]
LGTM

**Review by muchunjin — APPROVED** [review]
LGTM

**Pull request merged**